dyn_em/module_initialize_real.F

```diff
@@ -2983,19 +2983,6 @@ SUBROUTINE init_domain_rk ( grid &
       IF ( ( config_flags%sf_urban_physics == 1 ) .OR. ( config_flags%sf_urban_physics == 2 ) .OR. ( config_flags%sf_urban_physics == 3 ) ) THEN
          DO j = jts , MIN(jde-1,jte)
             DO i = its , MIN(ide-1,ite)
-              IF ( MMINLU == 'NLCD40' .OR. MMINLU == 'MODIFIED_IGBP_MODIS_NOAH') THEN
```

Comment:
In the PR commit message, would explain why this initialization is removed from the real program.

Reply:
will add to the commit message